aldjemy/geoalchemy2.py

```diff
@@ -0,0 +1,9 @@
+import geoalchemy2 as g2
+
+def get_geo_field(geometry_type, field):
```

Comment:
Actually you need place here check for geialchemy2 availability, and then provide simple importable function, instead of `try: except:` this module.